packaging/installer/kickstart.sh

```diff
@@ -1400,12 +1420,29 @@ try_static_install() {
   fi
 
   # Check status code first, so that we can provide nicer fallback for dry runs.
-  if ! check_for_remote_file "${NETDATA_STATIC_ARCHIVE_URL}"; then
+  if check_for_remote_file "${NETDATA_STATIC_ARCHIVE_URL}"; then
+    if [ -n "${INSTALL_VERSION}" ]; then
+      if [ "${SELECTED_RELEASE_CHANNEL}" = "stable" ]; then
+        netdata_agent="${NETDATA_STATIC_ARCHIVE_URL#"https://github.com/netdata/netdata/releases/download/v${INSTALL_VERSION}/"}"
+      else
+        netdata_agent="${NETDATA_STATIC_ARCHIVE_URL#"${NETDATA_TARBALL_BASEURL}/"}"
+      fi
+    else
+      if [ "${SELECTED_RELEASE_CHANNEL}" = "stable" ]; then
+        netdata_agent="${NETDATA_STATIC_ARCHIVE_URL#"https://github.com/netdata/netdata/releases/download/${latest}/"}"
+      else
+        netdata_agent="${NETDATA_STATIC_ARCHIVE_URL#"${NETDATA_TARBALL_BASEURL}/"}"
+      fi
+    fi
+  elif check_for_remote_file "${NETDATA_STATIC_ARCHIVE_OLD_URL}"; then
```

Comment:
```suggestion
  elif [ "${SYSARCH}" = "x86_64" ] && check_for_remote_file "${NETDATA_STATIC_ARCHIVE_OLD_URL}"; then
```

The ’old’ style static build filenames are only valid for 64-bit x86 systems, so we need to also only use this code _if_ the system CPU architecture is x86_64.